.projenrc.ts

```diff
@@ -232,5 +248,6 @@ rootProject.addTask('test:e2e', {
 rootProject.addTask('release',  {
   description: 'Release project',
 })
-
+rootProject.addGitIgnore(".jsii.tabl.json");
+rootProject.addPackageIgnore("!*.lit.ts");
```

Comment:
This one is already in the `fwkProject`. Do we need it at the top level or just inside of the framework project?

Reply:
removing from the root.